modules/simple-payments/simple-payments.php

```diff
@@ -64,7 +64,15 @@ function register_gutenberg_block() {
 		if ( $this->is_enabled_jetpack_simple_payments() ) {
 			jetpack_register_block( 'jetpack/simple-payments' );
 		} else {
-			Jetpack_Gutenberg::set_extension_unavailable( 'jetpack/simple-payments', 'missing_plan' );
+			$required_plan = ( defined( 'IS_WPCOM' ) && IS_WPCOM ) ? 'value_bundle' : 'jetpack_premium';
+			Jetpack_Gutenberg::set_extension_unavailable(
+				'jetpack/simple-payments',
+				'missing_plan',
+				array(
+					'required_feature' => 'simple-payments',
+					'required_plan' => ( defined( 'SHOW_BLOCK_UPGRADE_NUDGE' ) && SHOW_BLOCK_UPGRADE_NUDGE ) ? $required_plan : false
```

Comment:
I think there's a precedence of prefixing constants in Jetpack repo with `JETPACK_`.

Reply:
Another case where I thought using that prefix on WP.com could be confusing (where we actually _have_ to set the constant manually to enable the feature) :) And TBH, I see this as some kind of feature flag, i.e. hoping to drop it pretty soon :grimacing: I can add the prefix though, if you prefer.

Reply:
In my understanding anything shipping to self-hosted sites should be prefixed, just to stay on the safe side.  I'd expect someone from @Automattic/jetpack-crew to confirm in their review. :-)

Reply:
> In my understanding anything shipping to self-hosted sites should be prefixed

Makes sense, yeah. Just to clarify -- I hope to drop the feature flag before code freeze for the next JP release. But I'm willing to rename anyway, if you like.

Reply:
Added the `JETPACK_` prefix in 69d50cd7a